core/src/main/scala/kafka/metrics/KafkaMetricsReporter.scala

```diff
@@ -52,12 +52,11 @@ trait KafkaMetricsReporter {
 }
 
 object KafkaMetricsReporter {
-  val ReporterStarted: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
Sorry, I was looking at a different version of the code.  The existing version of the code in trunk does set it.  I suppose we should keep this as-is.